src/orbits/keplerian.jl

```diff
@@ -0,0 +1,524 @@
+using AstroLib: trueanom, kepler_solver
+using PhysicalConstants
+using Unitful, UnitfulAstro
+using KeywordCalls
+using Rotations
+
+# Domain specific unit conversions / Constants
+const G_nom = 2942.2062175044193 # Rsun^3/Msun/d^2
+const G_unit = PhysicalConstants.CODATA2018.G
+const MsunRsun_to_gcc = (1.0u"Msun/Rsun^3" |> u"g/cm^3").val
+
+"""
+    KeplerianOrbit(; kwargs...)
+
+Keplerian orbit parameterized by the basic observables of a transiting 2-body system.
+
+# Parameters
+* `period`/`P` -- The orbital period of the planet [d].
+* `t_0`/`t₀` -- The midpoint time of the reference transit [d].
+* `t_p`/`tₚ` -- The time of periastron [d].
+* `duration`/`τ` -- The transit duration [d].
+* `a` -- The semi-major axis [R⊙].
+* `R_planet`/`Rₚ` -- The radius of the planet [R⊙].
+* `R_star`/`Rₛ` -- The radius of the star [R⊙].
+* `rho_star`/`ρₛ` -- The spherical star density [M⊙/R⊙³].
+* `RpRs` -- The ratio of the planet radius to star radius.
+* `aR_star`/`aRₛ` -- The ratio of the semi-major axis to star radius.
+* `b` -- The impact parameter, bounded between 0 ≤ b ≤ 1.
+* `ecc` -- The eccentricity of the closed orbit, bounded between 0 ≤ ecc < 1.
+* `cos_omega`/`cos_ω` -- The cosine of the argument of periapsis.
+* `sin_omega`/`sin_ω` -- The sine of the argument of periapsis.
+* `incl` -- The inclination of the orbital plane relative to the axis perpendicular to the
+           reference plane [rad]
+* `omega`/`ω` -- The argument of periapsis [rad].
+* `Omega`/`Ω` -- The longitude of the ascending node [rad].
+* `M_planet`/`Mₚ` -- The mass of the planet [M⊙].
+* `M_star`/`Mₛ` -- The mass of the star [M⊙].
+"""
```

Comment:
Okay, let's start here- can you expand the docs explaining what the valid parameterizations are? For example, I tried looking up random orbital elements for Beta Pictoris b and making an orbit and I failed and I didn't really know which parameters I was missing or if it was a keyword name issue or something else. We need to be as clear as possible with how the users can parameterize the orbit, I think.

Reply:
Sure, this should follow the same [specs](https://docs.exoplanet.codes/en/latest/user/api/#exoplanet.orbits.KeplerianOrbit) as `exoplanet`. Does this work for you? The docs should be updated now as well